src/components/Transparency/MonthlyTotal.tsx

```diff
@@ -17,15 +18,31 @@ export type MonthlyTotalProps = React.HTMLAttributes<HTMLDivElement> & {
   invertDiffColors?: boolean
 }
 
-const RED = `Number--Red`
-const GREEN = `Number--Green`
+enum Color {
+  RED = 'Number--Red',
+  GREEN = 'Number--Green',
+}
+
+enum DetailsVisibility {
+  OVERVIEW = 'MonthlyTotal--Overview',
+  FULL = 'MonthlyTotal--Full',
+}
+
+const MAX_TAGS = 5
 
 export default React.memo(function MonthlyTotal({ title, monthlyTotal, invertDiffColors = false }: MonthlyTotalProps) {
   const t = useFormatMessage()
-  const [belowZeroColor, zeroOrOverColor] = invertDiffColors ? [GREEN, RED] : [RED, GREEN]
+  const [belowZeroColor, zeroOrOverColor] = invertDiffColors ? [Color.GREEN, Color.RED] : [Color.RED, Color.GREEN]
+  const [detailsVisibility, setDetailsVisibility] = useState(DetailsVisibility.OVERVIEW)
+
+  const click = () => {
```

Comment:
Rename this to `handleButtonClick` (or even more specific) so it's easier to see where this function is used 🙏